README.md

```diff
@@ -18,7 +18,7 @@
   - [🕹️ Interactive explorer](#️-interactive-explorer)
 - [🧭 Getting started](#-getting-started)
   - [🧩 1. Integrators: How to integrate with the taxonomy](#-1-integrators-how-to-integrate-with-the-taxonomy)
-    - [🗺️ Mapping to other taxonomies](#-mapping-to-other-taxonomies)
+    - [🗺️ Mapping to other taxonomies](#️-mapping-to-other-taxonomies)
```

Comment:
I added a hidden unicode character for this to work, I copied it from "setup and dependencies" link.